src/datasets/infra/repositories/DatasetsRepository.ts

```diff
@@ -45,10 +48,16 @@ export class DatasetsRepository extends ApiRepository implements IDatasetsReposi
       });
   }
 
+
+
   public async getDatasetCitation(datasetId: number, datasetVersionId: string): Promise<string> {
+      const queryParams: GetCitationQueryParams = {
+          includeDeaccessioned: true,
+
+      };
     return this.doGet(
       this.buildApiEndpoint(this.datasetsResourceName, `versions/${datasetVersionId}/citation`, datasetId),
-      true,
+      true,queryParams
```

Comment:
The `getDataset` method has the `includeDeaccessioned` flag as an argument, I think it should be the same for the `getDatasetCitation` method so the module user can decide to include deaccessioned or not

Reply:
Yes, that makes sense to keep it consistent, I added it as an argument.